README.md

```diff
@@ -37,8 +37,9 @@ This repository is made up of two separate Unity projects. If you would like to
 
 ## Features in XR Sample Project
 
+* [ARTableTop](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/xr_sample_project/Assets/SampleViewer/Samples/XRTableTop) - See how to configure the camera to visualize a tabletop map on a VR device and control the map with VR controllers and hand tracking.
```

Comment:
Just wanted to double check, since this is about ARTableTop, do we need to add AR just in case?

```suggestion
* [ARTableTop](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/xr_sample_project/Assets/SampleViewer/Samples/XRTableTop) - See how to configure the camera to visualize a tabletop map on an AR/VR device and control the map with AR/VR controllers and hand tracking.
```